code/bart_stability.R

```diff
@@ -30,13 +30,13 @@ get.features <- function(data){
     cov_matrix <- cbind(rbind(cov_q, cov_q_q), rbind(cov_q_q, cov_q))
     X <- mvrnorm(n = n, mu = rep(0, 2*q), Sigma = cov_matrix)
     y <- rowSums(X[, 1:q])
-    
-    
-    
+
+
+
   }
-  
+
   return(list(X=X, y=y))
-  
+
 }
 
 get_data <- function(dgp,n, q, rho=0.05, seed=0){
```

Comment:
Please generalize this function to include acic dgps, you can set a default value for q if that is a problem